manager/controllers/utils.go

```diff
@@ -96,16 +96,13 @@ type NestedFilterMap map[string]string
 
 var nestedFilters = NestedFilterMap{
 	"group_name":                                  "group_name",
-	"group_name][in":                              "group_name", // obsoleted, backward compatible
-	"system_profile][sap_system":                  "system_profile][sap_system",
-	"system_profile][sap_sids":                    "system_profile][sap_sids",
-	"system_profile][sap_sids][":                  "system_profile][sap_sids", // obsoleted, backward compatible
-	"system_profile][sap_sids][in":                "system_profile][sap_sids", // obsoleted, backward compatible
-	"system_profile][sap_sids][in][":              "system_profile][sap_sids", // obsoleted, backward compatible
-	"system_profile][ansible":                     "system_profile][ansible",
-	"system_profile][ansible][controller_version": "system_profile][ansible][controller_version",
-	"system_profile][mssql":                       "system_profile][mssql",
-	"system_profile][mssql][version":              "system_profile][mssql][version",
+	"system_profile][sap_system":                  "(ih.system_profile->'workloads'->'sap'->>'sap_system')",
+	"system_profile][sap_sids":                    "(ih.system_profile->'workloads'->'sap'->>'sap_sids')",
```

Comment:
I think it should `sids` and not `sap_sids`. 

See https://github.com/RedHatInsights/insights-host-inventory/blob/master/swagger/system_profile.spec.yaml#L1024-L1051

Reply:
fixed, thanks.